jenkinsfiles/ginkgo-kernel.Jenkinsfile

```diff
@@ -201,6 +201,7 @@ pipeline {
                 KUBECONFIG="${TESTDIR}/vagrant-kubeconfig"
                 FAILFAST=setIfLabel("ci/fail-fast", "true", "false")
                 CONTAINER_RUNTIME=setIfLabel("area/containerd", "containerd", "docker")
+                HOST_FIREWALL=setIfLabel("ci/host-firewall", "1", "0")
```

Comment:
- To be consistent with the previous setting `ci/fail-fast`, maybe `"true"` and `"false"` instead? UPDATE: Ehrm `KUBEPROXY` is using `0|1` :-/
- AFAIK, `ginkgo-kernel.Jenkinsfile` is used by net-next/4.19. Do we want to enable the host-fw e.g. for GKE (`ginkgo-gke.Jenkinsfile`)? Maybe it's possible to create a common jenkinsfile?

Reply:
> * To be consistent with the previous setting `ci/fail-fast`, maybe `"true"` and `"false"` instead? UPDATE: Ehrm `KUBEPROXY` is using `0|1` :-/

I've made a note to send a follow up PR to make the environment variable values consistent.

> * AFAIK, `ginkgo-kernel.Jenkinsfile` is used by net-next/4.19. Do we want to enable the host-fw e.g. for GKE (`ginkgo-gke.Jenkinsfile`)? Maybe it's possible to create a common jenkinsfile?

I'm a bit loss in all the Jenkinsfiles to be honest. I think I also need to update `ginkgo.Jenkinsfile`. Enabling the host firewall on GKE would require a bit more work though, mostly because the host firewall cannot run with GKE's default options (per-endpoint routes).